sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventProcessorClientBuilder.java

```diff
@@ -237,6 +238,16 @@ public EventProcessorClientBuilder configuration(Configuration configuration) {
         return this;
     }
 
+    /**
+     * Sets the amqp client options.
+     * @param amqpClientOptions The amqp client options.
+     * @return The updated {@link EventProcessorClientBuilder} object.
+     */
+    public EventProcessorClientBuilder amqpClientOptions(AmqpClientOptions amqpClientOptions) {
```

Comment:
I note that on the HTTP side, where have an analogous `HttpClientOptions`, because of historical choices we tend to have the builder methods just be named `clientOptions` (and this is what we ask for in the design guidelines too). The same should apply here (whilst still being specific about the `AmqpClientOptions` parameter type). It gives a better feeling of consistency across the libraries. // @srnagar thoughts?

Reply:
Got it. I'll change `AmqpClientOptions` extend `ClientOptions` and use it in `clientOptions()` method.

Reply:
Agree with Jonathan that we should use `clientOptions()` method in builder and accept the AmqpClientOptions type here similar to what we do for Http client builders.

Reply:
Just to be clear - I'm not necessarily advocating for `AmqpClientOptions` to extend `ClientOptions` - we'll have to review if that makes sense in the AMQP context.

Reply:
Hi @JonathanGiles , I've fixed it.